data/sources/46_overlay-maps.yaml

```
@@ -141,10 +141,10 @@
 - props:
     parent: "mw"
     box:
-      - [11.667506, 48.266657]
-      - [11.671514, 48.266191]
-      - [11.671084, 48.264587]
-      - [11.667094, 48.265051]
+      - [11.667629, 48.267174]
+      - [11.671599, 48.266688]
+      - [11.670956, 48.264145]
+      - [11.666920, 48.264598]
```

Corrected the coordinates as by suggestion
```suggestion
      - [11.667617801785777, 48.26719015728]
      - [11.67160725799323, 48.26672188711042]
      - [11.670921198214224, 48.264131842720005]
      - [11.66693174200677, 48.26460011288958]
```